froide/foirequest/models/attachment.py

```diff
@@ -163,6 +163,7 @@ class FoiAttachment(models.Model):
     is_converted = models.BooleanField(_("Is converted"), default=False)
     timestamp = models.DateTimeField(null=True, default=timezone.now)
     pending = models.BooleanField(default=False)
+    has_been_moderated = models.BooleanField(_("Has been moderated"), default=False)
```

Comment:
I would rather see this as `is_moderated` to keep in line with `is_converted` and `is_redacted`.